test/test_htcondor_parser.py

```diff
@@ -53,20 +55,20 @@ def test_parse_line_no_cputmult(self):
 
         cases = {}
         for line, value in zip(lines, values):
-            cases[line] = dict(zip(keys, value))
+            cases[line] = dict(list(zip(keys, value)))
```

Comment:
The `list` (and the `.key()`) calls in this file changeset are superfluous. See previous comments for explanations.

Reply:
Make sense @maxfischer2781. Added this in [daecf80](https://github.com/apel/apel/pull/354/commits/daecf8087f5404c800245f05929f65f1aa0800ab)